src/graphnet/models/model.py

```diff
@@ -164,6 +183,9 @@ def predict_as_dataframe(
             additional_attributes = []
         assert isinstance(additional_attributes, list)
 
+        if prediction_columns is None:
+            prediction_columns = self.prediction_columns
```

Comment:
These are not set for all instances of `Model` (currently, in this PR, this is only done for `StandardModel` which inherits from `Mode`), so either should we do that, otherwise we shouldn't presuppose the existence of this attribute.